docs/access-management/rbac.md

```diff
@@ -0,0 +1,54 @@
+---
+products: oss-enterprise, cloud-teams
+---
+
+# Role Based Access Control (RBAC)
+
+Role Based Access Control allows a user with Administrative access to apply roles to users, granting different levels of permission within an organization or workspace. 
+
+## Workspace Resource Roles
+A user can have at most one role of this kind per workspace. Permissions are scoped to the specific workspace in which the user has this role.
+
+| Permissions           | WorkspaceReader    | WorkspaceEditor    | WorkspaceAdmin    |
+| ---------------------- | :--------: | :--------:| :--------: |
+| **ReadWorkspace**<br />-List the connections in a workspace<br />-Read individual connections<br />-Read workspace settings (data residency, users, connector versions, notification settings) | X | X | X |
+| **Update Connection**<br />-Start/cancel syncs<br />- Modify a connection, including name, replication settings, normalization, DBT<br />- Delete a connection<br />- Create/Update/Delete connector builder connectors |  | X | X |
+| **UpdateWorkspace**<br />- Update workspace settings (data residency, users, connector versions, notification settings)<br />- Modify workspace connector versions | |  | X |
+
+## Organization Resource Roles
+
+A user can have at most one role of this kind per organization. Permissions are scoped to the given organization for which the user has this role, and any workspaces within.
+
+| Permissions           | OrganizationMember | OrganizationReader | OrganizationEditor |OrganizationAdmin |
+| :---------------------- | :--------: | :--------: | :--------: |:--------: |
+| **ReadOrganization**<br />- Read individual organizations | X | X | X | X |
+| **CreateWorkspace**<br />- Create new workspace within a specified organization<br />- Delete a workspace | | | X | X |
+| **UpdateOrganization**<br />- Modify organization settings, including billing, PbA, SSO<br />- Modify user roles within the organization | |  |  | X |
+
+## Instance Resource Roles
+
+At the instance level, a user may have InstanceAdmin role. Permissions are valid for all workspaces and all organizations. This user, therefore would have the following permissions as `InstanceAdmin`: 
+
+**ReadWorkspace**
```

Comment:
I don't think we need to list the roles for instance admins. I think we should instead explain that the instance admin is only relevant for self-managed enterprise, and corresponds to the first user who logs into Airbyte.

Reply:
So I would make this section much shorter :)

Reply:
Good! I was hoping someone would say this. I included it all to be thorough but, now with more info about when instance admins are created, I think this level of detail is unneeded and could even be confusing.